src/content/reference/react/isValidElement.md

```diff
@@ -90,39 +90,39 @@ console.log(isValidElement([<div />, <div />])); // false
 console.log(isValidElement(MyComponent)); // false
 ```
 
-It is very uncommon to need `isValidElement`. It's mostly useful if you're calling another API that *only* accepts elements (like [`cloneElement`](/reference/react/cloneElement) does) and you want to avoid an error when your argument is not a React element.
+`isValidElement` sangat jarang diperlukan. Ini sangat berguna jika Anda memanggil API lain yang *hanya* menerima elemen (seperti halnya [`cloneElement`](/reference/react/cloneElement)) dan Anda ingin menghindari kesalahan saat argumen Anda bukan elemen React.
 
-Unless you have some very specific reason to add an `isValidElement` check, you probably don't need it.
+Kecuali Anda memiliki alasan yang sangat spesifik untuk menambahkan pemeriksaan `isValidElement`, Anda mungkin tidak membutuhkannya.
 
 <DeepDive>
 
-#### React elements vs React nodes {/*react-elements-vs-react-nodes*/}
+#### Elemen React vs *node* React {/*react-elements-vs-react-nodes*/}
 
-When you write a component, you can return any kind of *React node* from it:
+Saat Anda menulis sebuah komponen, Anda dapat mengembalikan *node React* apa pun darinya:
 
 ```js
 function MyComponent() {
-  // ... you can return any React node ...
+  // ... anda dapat mengembalikan node React apapun ...
 }
 ```
 
-A React node can be:
+Node React dapat berupa:
 
-- A React element created like `<div />` or `createElement('div')`
-- A portal created with [`createPortal`](/reference/react-dom/createPortal)
-- A string
-- A number
-- `true`, `false`, `null`, or `undefined` (which are not displayed)
-- An array of other React nodes
+- Elemen React yang dibuat seperti `<div />` atau `createElement('div')`
+- Portal yang dibuat dengan [`createPortal`](/reference/react-dom/createPortal)
+- Sebuah *string*
+- Sebuah Angka
+- `true`, `false`, `null`, atau `undefined` (yang tidak ditampilkan)
+- Array *node* React lainnya
```

Comment:
Need suggestion from @mazipan @resir014 @r17x @zainfathoni how to improve this sentence.

```suggestion
- Array *node* React lainnya
```

Reply:
This already good from me.